src/tictactoe/lib/tictactoe/GameJsonRepository.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Tictactoe.GameJsonRepository do
+  @callback save(Map) :: String
+  @callback load(String) :: Map
+
+  @directory "./../../data"
```

Comment:
That's a nice one! Let's think together about it
Since Json storage is a temporary solution, I would like to leave repository method calls as one-liners.
So the only change that would be needed when switching the store is:
`alias Tictactoe.GameJsonRepository, as: Repo`
to
`alias Tictactoe.GameDocumentDBRepository, as: Repo`

At the same time I indeed need to solve the issue of piling test data.